app/mailers/notifier.rb

```diff
@@ -0,0 +1,10 @@
+class Notifier < ActionMailer::Base
+  def statistic statistic
+    @daily_statistic = statistic
+
+    mail subject: t('digest.email_header.subject', date: @daily_statistic.for_today),
+         to:      User.all.map {|user| user.email},
+         bcc:     ENV['BCC'],
+         from:    t('digest.email_header.from')
+  end
```

Comment:
@vlasiak, Take a look at [the suggestions to align parameters](https://github.com/bbatsov/ruby-style-guide#no-double-indent) (no need to align them "by value").